beep/tests/test_maccor_to_biologic_mb.py

```diff
@@ -159,34 +246,428 @@ def test_discharge_current_step_conversion(self):
+            "    </FileFormatVersion>"
+            "    <ProcDesc>"
+            "      <desc></desc>"
+            "    </ProcDesc>"
+            "  </header>"
+            "  <ProcSteps>"
+            "    <TestStep>"
+            "      <StepType>  Rest  </StepType>"
+            "      <StepMode>        </StepMode>"
+            "      <StepValue></StepValue>"
+            "      <Limits/>"
+            "      <Ends>"
+            "        <EndEntry>"
+            "          <EndType>StepTime</EndType>"
+            "          <SpecialType> </SpecialType>"
+            "          <Oper> = </Oper>"
+            "          <Step>002</Step>"
+            "          <Value>03:00:00</Value>"
+            "        </EndEntry>"
+            "      </Ends>"
+            "      <Reports>"
+            "        <ReportEntry>"
+            "          <ReportType>StepTime</ReportType>"
+            "          <Value>00:00:30</Value>"
+            "        </ReportEntry>"
+            "      </Reports>"
+            "      <Range>4</Range>"
+            "      <Option1>N</Option1>"
+            "      <Option2>N</Option2>"
+            "      <Option3>N</Option3>"
+            "      <StepNote></StepNote>"
+            "    </TestStep>"
+            "    <TestStep>"
+            "      <StepType> Charge </StepType>"
+            "      <StepMode>Current </StepMode>"
+            "      <StepValue>1.0</StepValue>"
+            "      <Limits/>"
+            "      <Ends>"
+            "        <EndEntry>"
+            "          <EndType>StepTime</EndType>"
+            "          <SpecialType> </SpecialType>"
+            "          <Oper> = </Oper>"
+            "          <Step>003</Step>"
+            "          <Value>00:00:01</Value>"
+            "        </EndEntry>"
+            "      </Ends>"
+            "      <Reports>"
+            "        <ReportEntry>"
+            "          <ReportType>StepTime</ReportType>"
+            "          <Value>::.01</Value>"
+            "        </ReportEntry>"
+            "      </Reports>"
+            "      <Range>A</Range>"
+            "      <Option1>N</Option1>"
+            "      <Option2>N</Option2>"
+            "      <Option3>N</Option3>"
+            "      <StepNote>resistance check</StepNote>"
+            "    </TestStep>"
+            "    <TestStep>"
+            "      <StepType>  Do 1  </StepType>"
+            "      <StepMode>        </StepMode>"
+            "      <StepValue></StepValue>"
+            "      <Limits/>"
+            "      <Ends/>"
+            "      <Reports/>"
+            "      <Range></Range>"
+            "      <Option1></Option1>"
+            "      <Option2></Option2>"
+            "      <Option3></Option3>"
+            "      <StepNote></StepNote>"
+            "    </TestStep>"
+            "    <TestStep>"
+            "      <StepType> Charge </StepType>"
+            "      <StepMode>Voltage </StepMode>"
```

Comment:
I think that if the limit is voltage based, StepMode needs to be `Current` or vise-versa